### PR TITLE
christin

### PR DESCRIPTION
**Pull request by chmette**
please chck my code and maybe give a hint in number 3. im a bit stuck

### CODE REVIEW

File: christin/index.js [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+let volume1 = new Volume(3, 6);
+console.log(volume1.getVolume());
+//----------------------------------------###------------------
+
+//*Rewrite the following code to use the "class" syntax.
+
+class Clock {
+  constructor({ template }) {
+    let timer;
```
Comment (ciamac-da) [inline christin/index.js]: Task 3 ==> There should be this.timer; instead of let timer;

File: christin/index.js [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+  }
+
+  stop() {
+    this.stop = function () {
+      clearInterval(timer);
+    };
+    return this.stop;
+  }
+
+  start() {
```
Comment (ciamac-da) [inline christin/index.js:73]: Task 3 ==> start method is wrong 
- First of all you need this.render() to get access to the render method
- Then you have to make a setInterval method for the timer like this ==>
 this.timer = setInterval(()=>{
        this.render()
    }, 1000)

File: christin/index.js [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    let output = this.template
+      .replace("h", hours)
+      .replace("m", mins)
+      .replace("s", secs);
+
+    console.log(output);
+    return output;
+  }
+
+  stop() {
```
Comment (ciamac-da) [inline christin/index.js:69]: Task 3 ==> All you need inside of stop method is clearInterval(this.timer)

File: christin/index.js [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+        timer = setInterval(render, 1000);
+      };
+    };
+    return this.start;
+  }
+}
+
+let clock = new Clock({ template: "h:m:s" });
+clock.start();
+
```
Comment (ciamac-da) [inline christin/index.js]: Task 3 ==> Now timer works, if you wanna stop the timer after some time you can add setTimeout method after clock.start()
setTimeout(() => clock.stop(), 4000);   <== in this case we stop the timer after 4seconds

**Review by ciamac-da — CHANGES_REQUESTED** [review]

File: christin/index.js (hunk 2 above)
Comment (chmette) [reply christin/index.js:73]: i added changes, please check

**Review by ciamac-da — APPROVED** [review]
🥇

**Pull request merged**